PWGCF/FemtoDream/femtoDreamPairTaskTrackTrack.cxx

```diff
@@ -165,6 +197,10 @@ struct femtoDreamPairTaskTrackTrack {
         continue;
       }
       trackHistoPartOne.fillQA(part);
+      if constexpr (isMC) {
+        trackHistoPartOneMC.fillQA<true>(partsMC.iteratorAt(part.index()));
+        trackHistoPartOneMC.fillQAMC(part, partsMC.iteratorAt(part.index()));
```

Comment:
This is the rigidity that not having an `MCTrackLabels` like table imposes
Please, reconsider it
You are imposing the same ordering on two different tables